content/html/concepts/forms/forms.md

```diff
@@ -16,12 +16,12 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-In HTML, forms are used for gathering and sending user input to an external source (like a [server](https://www.codecademy.com/resources/docs/general/server)) for processing. This is done with the [`<form>`](https://www.codecademy.com/resources/docs/html/elements/form) tag, which contains various labels and [inputs](https://www.codecademy.com/resources/docs/html/elements/input).
+In HTML, forms are used for gathering and sending user input to an external source (like a [server](https://www.codecademy.com/resources/docs/general/server)) for processing. This is done with the [`<form>`](https://www.codecademy.com/resources/docs/html/elements/form) element, which contains various [labels](https://www.codecademy.com/resources/docs/html/elements/label) and [inputs](https://www.codecademy.com/resources/docs/html/elements/input).
 
-## Example
-
-The following example
+## Syntax
 
+The following example shows what an HTML form would typically look like with a `<form>` element that features `<input>` elements and its corresponding `<label>` elements:
```

Comment:
```suggestion
The following example shows what an HTML form that features `<input>` elements:
```

I actually don't see any labels in the example. Did you mean to add them to the example?